src/mui/field/PhoneField.spec.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import assert from 'assert';
+import { shallow } from 'enzyme';
+import PhoneField, { localPhoneFormat } from './PhoneField';
+
+describe.only('<PhoneField />', () => {
+  // localPhoneFormat tests
```

Comment:
instead of having useless comments you could separate test cases in 2 different describes :)